domains/oops.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "finnickguo",
+        "email": "finnickguo@gmail.com"
+    },
+    "record": {
+        "CNAME": "150.109.115.77"
```

```suggestion
        "A": "150.109.115.77"
```
Invalid CNAME. What you want is likely a A record.